src/components/BasketballCourt/index.tsx

```diff
@@ -20,8 +23,12 @@ const BasketballCourt = () => {
       minHeight={400}
       justifyContent="center"
       alignItems="center"
+      flexDirection="column"
       margin="auto"
     >
+      <Heading as="h2" transform="translateY(37px)" fontWeight="medium" fontSize="20px">
+        {courtName}
+      </Heading>
```

Comment:
This implementation is different from Beryl's. Please be aware of that. You both modified this part.